src/common/AudioStream.cpp

```diff
@@ -196,4 +196,13 @@ ResultWithValue<FrameTimestamp> AudioStream::getTimestamp(clockid_t clockId) {
     }
 }
 
+void AudioStream::calculateDefaultDelayBeforeCloseMillis() {
```

Comment:
Are we guaranteed that calculateDefaultDelayBeforeCloseMillis() runs before the user gets an AudioStreamPointer? If not, setDelayBeforeCloseMillis() may be overriden by calculateDefaultDelayBeforeCloseMillis(). Can we add some comments about the timing windows?